### ISSUE TITLE
Update format of github --> github.io copying

### ISSUE THREAD

**Issue by risserlin** [bioinformaticsdotca/Pathways_2019#2]
If you use relative paths in the github markdown file and then copy over to github.io all relative paths are not found.  
Need to change the structure of the output web page so that they respect relative paths so we don't need to check all images and links after every replication

**Comment by mbrazas** [comment]
Agreed. Image embedding is supposed to be done through an "Issue" rather than uploading a file.
https://github.com/RWTH-EBC/AixLib/wiki/How-to:-Add-images-to-the-Wiki
